### DISCUSSION TITLE
Two months bills

### DISCUSSION THREAD

**Discussion by M4RC02U1F4A4** [firefly-iii/firefly-iii#4410] [General]
**Description**
It would be nice to have the option repeat the bills every 2 month, as there is already the option to do it every 4, 6, 12 months 

**Solution**
Add "repeat every 2" months in the menu `Repeats` on the Bills page

**Additional context**
Here in Italy we have some bills that are paid every 2 month, like electricity for example

**Comment by JC5** [comment]
https://docs.firefly-iii.org/firefly-iii/advanced-concepts/bills/

> If you enter a number in the "skip" field, the bill will be automatically skipped every X times; a bill that arrives every 3 months can be entered by filling in "2".
